test/functional/wallet_backup.py

```diff
@@ -140,6 +140,23 @@ def restore_wallet_existent_name(self):
         assert_raises_rpc_error(-36, error_message, node.restorewallet, wallet_name, backup_file)
         assert wallet_file.exists()
 
+    def test_pruned_wallet_backup(self):
+        self.log.info("Test loading backup on a pruned node with backup was created close to the prune height of the restoring node")
+        node = self.nodes[3]
+        self.restart_node(3, ["-prune=1", "-fastprune=1"])
+        # We need a few more blocks so we can actually get above an realistic
```

Comment:
nit:
Assert that the chain height is 214, because the test assume so?
```suggestion
        # Ensure the chain tip is at height 214, because this test assume it is.
        assert_equal(node.getchaintips()[0]["height"], 214)
        # We need a few more blocks so we can actually get above an realistic
```

Reply:
added